_posts/Prakash-sa/2022-10-06-clicky-turtle-hack.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: Prakash-sa
+title: Clicky Turtle Hack
+---
+
+
+It will ask for the input and make polygon with the turtle.
+
+
+https://trinket.io/pygame/565ae1e960
```

Comment:
This will show up as just text. I believe you'll want the embed code, which you get via the Share button.

Reply:
Thanks for mentioning it. I have added the embedded link.